app/assets/stylesheets/helpers/_emergency-publishing-notifications.scss

```diff
@@ -32,14 +32,11 @@
   a {
     color: #fff;
 
-    &.right {
+    &.more-information {
       display: block;
-      padding: 0;
 
       @include media(tablet) {
-        padding: 0 30px 0 0;
-        margin-top: (25 / 19) * -1em;
-        text-align: right;
+        padding: 0 30px 0 30px;
```

Comment:
Yep. I'm not sure if this project is linted, but we should lint for one behaviour or the other (I'm impartial to which)